### ISSUE TITLE
Upgrade documentation for the cron and timer hives

### ISSUE THREAD

**Issue by Zykino** [muesli/beehive#293]
I just saw there are Cron and Time hives and they look too be identical.
The descriptions should better state that Cron is recurring and Time one time only ? The Time hive may need a "once" or "one time only" in the description to make the difference really clear.

And if possible they should be near (Time once, Time recurring (cron)) so user will see both at once and choose whichever they need.

**Comment by rubiojr** [comment]
:+1: The intention of both hives hasn't been clear to me either. I've been thinking we had two different implementations that do the same thing for a while.

**Comment by rubiojr** [comment]
@muesli, would it help to rename that hive to "Once", "On This Date" or something like that?

**Comment by Zykino** [comment]
I heard you are reworking the admin interface (potentially adding categories) but I think that it is quite important that since this two hives are so close (utility, logo, input, descriptions) that they need to share part of the name and be differentiated on an other part. That way they will be next to each other in the list and as a user I will ask myself which one I need right now. Except if you intent to randomize the hives order…

Something like:
Time (Once)
Time (Recurrent)

**Comment by muesli** [comment]
I agree, we should clean that up and combine those two Hives into one, which then offers multiple event types.

We should probably also drop the underlying cron library and switch to a maintained and better tested one: https://github.com/robfig/cron
